src/components/Header/Modal/HeaderModal.tsx

```diff
@@ -1,49 +1,93 @@
 "use client";
-import {
-  ChildrenWrapper,
-  HeaderItem,
-  ItemWrapper,
-  Modal,
-  NavList,
-  Navbar,
-} from "./HeaderModal.styles";
-import { ISection } from "@/utils/interfaces";
-import { useState } from "react";
-
-const NavItem = ({ item }: { item: { fields: ISection } }) => {
-  const { children, path, name } = item.fields;
 
-  if (!children) {
-    return <HeaderItem href={path}>{name}</HeaderItem>;
-  }
+import { ISection } from "@/utils/interfaces";
 
-  return (
-    <ItemWrapper>
-      <HeaderItem href={path}>{name}</HeaderItem>
-      <ChildrenWrapper>
-        {children.map((child, key) => (
-          <NavItem key={key} item={child} />
-        ))}
-      </ChildrenWrapper>
-    </ItemWrapper>
-  );
-};
+import { Sheet, SheetTrigger, SheetContent } from "@/components/ui/sheet";
+import { Menu } from "lucide-react";
+import Link from "next/link";
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from "@/components/ui/accordion";
+import { macroThemes } from "@/utils/constants";
+import { Icon } from "@/components/Icon/Icon";
+import { usePathname } from "next/navigation";
 
 const HeaderModal = ({ content }: { content: { fields: ISection }[] }) => {
-  const [retracted, setRetracted] = useState<boolean>(true);
+  const pathname = usePathname();
 
   return (
-    <Modal retracted={retracted} setRetracted={setRetracted}>
-      <Navbar>
-        <NavList>
+    <Sheet>
+      <SheetTrigger asChild>
+        <div className="flex items-center">
+          <Menu className="h-5 w-5 text-green-900" />
+        </div>
+      </SheetTrigger>
+      <Link href="/" className="flex items-center gap-2">
+        <img src="/logo-DNE.png" alt="Logo" className="w-[99px] h-[47px]" />
+      </Link>
+
+      <SheetContent
+        side="left"
```

Comment:
Usually on some projects I've seen this modal takes all the page when they're built like this one

```suggestion
        side="top"
```